### ISSUE TITLE
strange loss and presion 

### ISSUE THREAD

**Issue by 3-leaves-grass** [ZhaoJ9014/face.evoLVe.PyTorch#103]
@ZhaoJ9014  I used the default parameters in your config and trained model in the dataset（Asian-Celeb 93,979 ids with 2,830,146 aligned images)
![QQ截图20191205102511](https://user-images.githubusercontent.com/37999994/70198737-a2870d80-174a-11ea-9fb5-9f8c6b8d47ea.png)
，however， the loss was still high and Precision was 0 after epoch 72，what should I do ？ is this normal？or is it the matter of parameters in config？

**Comment by ShaoLingXie** [comment]
hi, i also meet this problem, do u solve it?

**Comment by 3-leaves-grass** [comment]
yeah， making the value of s smaller can solve it

**Comment by ppkw2000** [comment]
Hello, I have this problem as well. What is the value of s ? @3-leaves-grass

**Comment by pppnkn** [comment]
Hello, I have the same problem as well. Could you please tell me what is the value of s ? @3-leaves-grass  thank you.

**Comment by 3-leaves-grass** [comment]
the parameter s is the norm of  input feature(default 64) , can find it in metrics.py @pppnkn

**Comment by XinWangg** [comment]
Hello, I have the same problem as well. Could you please tell me how to set the value s? what is the value of s you choose?
